src/ocm/components/clusterDetail/AssistedInstallerDetailCard.tsx

```diff
@@ -37,17 +37,16 @@ type AssistedInstallerDetailCardProps = {
 
 const getErrorStateActions = () => {
   const errorStateActions: React.ReactNode[] = [];
-  if (!isSingleClusterMode()) {
-    errorStateActions.push(
-      <Button
-        key="cancel"
-        variant={ButtonVariant.secondary}
-        component={(props) => <Link to={OCM_CLUSTER_LIST_LINK} {...props} />}
-      >
-        Back
-      </Button>,
-    );
-  }
+  errorStateActions.push(
```

Comment:
We can now define the list statically, so it doesn't need to be a function anymore.

Reply:
Yes 10x, you are right! I overlooked this part

Reply:
Addressed in bfac0e1b807c523bfa47cb2c1f3803deec4b8623